src/conn/conn_las.c

```diff
@@ -0,0 +1,400 @@
+/*-
+ * Copyright (c) 2014-2015 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
```

Comment:
Can you please move this file to `cache/cache_las.c`?

I'd like to reduce the things in `conn` -- the current grouping doesn't make a lot of sense to me.